pkg/controller/model_binding_pair.go

```diff
@@ -345,6 +385,20 @@ func buildModelBindingPair(mbPair *types.ModelBindingPair) *types.ModelBindingPa
 					}
 				}
 			}
+
+			genericComponents := mbPair.Model.Spec.GenericComponents
+			if genericComponents != nil {
+				for _, generic := range genericComponents {
+					_, found := compNames[generic.Name]
+					if found {
+						for _, connection := range generic.Connections {
+							createRemoteRestConnections(mbPair, mc, connection.Rest, namespace.Name)
+							envVars := getRestConnectionEnvVars(mbPair, connection.Rest, generic.Name)
+							genericcomp.AddEnvVars(mc, generic.Name, envVars)
```

Comment:
For the other components in this loop, the method being called is `UpdateEnvVars`.  Should this also be update, or should they all be add?

Reply:
I will change to UpdateEnvVars for consistency

Reply:
Fixed